### DISCUSSION TITLE
Is it possible to use lessmsi instead of using msiexec?

### DISCUSSION THREAD

**Discussion by rsp4jack** [microsoft/vcpkg#30772] [Q&A] [answered]
msiexec does not run correctly on my PC. Is it possible to use lessmsi for vcpkg_find_acquire_program？

**Comment by Osyotr** [comment]
Yes. You can use `kinectsdk2` as an example:
https://github.com/microsoft/vcpkg/blob/dafff8267e0685baa86140e5a5dcf4d9d7383592/ports/kinectsdk2/vcpkg.json#L9-L13
https://github.com/microsoft/vcpkg/blob/dafff8267e0685baa86140e5a5dcf4d9d7383592/ports/kinectsdk2/portfile.cmake#L21-L29

**Reply by rsp4jack** [reply]
I mean, can I use lessmsi in:

https://github.com/microsoft/vcpkg/blob/203383666e2422ed31e1faebe6efa6e306bd126d/scripts/cmake/vcpkg_find_acquire_program.cmake#L655-L663

**Reply by Osyotr** [reply, marked as answer]
Ahh I see. No, I don't think that's possible without patching `vcpkg_find_aquire_program` because `msiexec` name is hardcoded.
The fix would be pretty straightforward: add an entry for `lessmsi` in `vcpkg_find_acquire_program`, call `vcpkg_find_acquire_program(lessmsi)` here and replace `msiexec` with `${lessmsi}`.
